spec/support/fake_heroku.rb

```diff
@@ -36,8 +36,16 @@ def self.has_created_app_for?(environment, flags = nil)
   end
 
   def self.has_configured_vars?(remote_name, var)
-    File.foreach(RECORDER).any? do |line|
-      line =~ /^config:add #{var}=.+ --remote #{remote_name}\n$/
-    end
+    commands_ran =~ /^config:add #{var}=.+ --remote #{remote_name}\n/
+  end
+
+  def self.has_setup_pipeline_for?(app_name)
+    commands_ran =~ /^pipelines:create -a #{app_name}/ &&
+      commands_ran =~ /^pipelines:add -a #{app_name}-staging #{app_name}/ &&
+      commands_ran =~ /^pipelines:add -a #{app_name}-production #{app_name}/
```

Comment:
Will wait till the other branch gets merged before fixing this (not sure if this Fake changes again).